src/lib/templates/edge-function-error.html

```diff
@@ -0,0 +1,289 @@
+<!DOCTYPE html>
```

Comment:
What would it take for us to use the same template for both types of functions? Are they all that different? 🤔

Reply:
There are some differences right now in some of the texts, and some of the links... I also saw some things in the normal functions template that I'm pretty sure we won't use for the edge functions template, but that I'm not sure about if we use it still for normal functions. So I felt unsure as to combine the two. 

I could pass the differences as arguments, but something about that feels off a bit as well because the amount of arguments will grow quite a bit.

Reply:
I've asked some input from the team, if it's all good to clear out some old stuff we don't need anymore I'll merge the two :)